ansible/configs/sap-smart/post_software.yml

```diff
@@ -0,0 +1,198 @@
+- name: Step 00xxxxx post software
```

Comment:
What is this section for targeting `support` hosts? If it is not needed and not doing anything, we shoudl start with the first actual play. The debug tasks can still be included in that play if you want.

Reply:
This is coming from the original sample config I used in the past to create mines. Just removed this.